### ISSUE TITLE
Incrementally editing __done_exclude (as list?)

### ISSUE THREAD

**Issue by Mekk** [franciscolourenco/done#99]
After using done for some time, I find maintaining `__done_exclude` unpleasant. It simply grows long. Mostly it's because I happen to spawn GUI apps from the terminal, and -  for example - when my `emacsclient -cn` ends it's job after an hour or two, fish gladly notifies me.

So it would be nice if I could „add next exception without touching those already existing”. For example by sth like
```
set -Ua __done_exclude  emacsclient
set -Ua __done_exclude '(scite|gedit)'
```
(result: all exceptions which were present, are still there, and I just added two more)

**Comment by franciscolourenco** [comment]
This has been implemented and going to be available in v1.17.0

**Comment by Mekk** [comment]
Thank you.
